app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -1996,6 +2034,31 @@ class MessagingManager @Inject constructor(
         WebsocketManager.start(context)
     }
 
+    private fun processScreenBrightness(data: Map<String, String>): Boolean {
+        val command = data[COMMAND]
+        val contentResolver = context.contentResolver
+        val success = Settings.System.putInt(
+            contentResolver,
+            if (data[MESSAGE].toString() == COMMAND_SCREEN_BRIGHTNESS_LEVEL)
+                Settings.System.SCREEN_BRIGHTNESS
+            else
+                Settings.System.SCREEN_BRIGHTNESS_MODE,
+            if (data[MESSAGE].toString() == COMMAND_SCREEN_BRIGHTNESS_LEVEL)
+                when {
+                    command!!.toInt() < 0 -> 0
+                    command.toInt() > 255 -> 255
+                    else -> command.toInt()
+                }
```

Comment:
Kotlin has a nice function for this: [`coerceIn`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/coerce-in.html)

```suggestion
                command!!.toInt().coerceIn(0..255)
```